integration_tests/generics_01.py

```diff
@@ -7,4 +7,4 @@ def f(x: T, y: T) -> T:
 
 print(f(1,2))
 print(f("a","b"))
-print(f("c","d"))
+print(f("c","d"))
```

Comment:
This is just a small thing, but if the only change is just a white space, let's not do it in this PR, you can fix that by:
```
git checkout main integration_tests/generics_01.py
```
Then update tests too, it will remove the change in the reference result for this.